app/models/user.rb

```diff
@@ -9,4 +9,6 @@ class User < ApplicationRecord
   has_many :posts
   has_many :comments, dependent: :destroy
   has_many :likes, dependent: :destroy
+  has_many :friendships
+  has_many :inverse_friendships, class_name: 'Friendship', foreign_key: 'friend_id'
```

Comment:
**Kindly work on these changes:**
- User can reject a friendship request ( [screenshot ](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/final_project/milestone5/accept_reject_friendship.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/49736080/)).

**[OPTIONAL]**
**- Add your personal information and a license into your README file. Would be great to have fully detailed this file to improve your professionalism once the recruiters come to your repositories.**